apps/sms/views/shared/test/unit/contacts_test.js

```diff
@@ -789,7 +789,8 @@ suite('Contacts', function(done) {
     // to the console
     test('({}, ...)', function(done) {
       Contacts.findBy({}).then(function(contacts) {
-        assert.equal(contacts, null);
+        assert.ok(Array.isArray(contacts));
```

Comment:
when contacts return an empty array, it would evaluate to false.
so if i use assert.isTrue it would still give me the error in contact_test.js